test/mailer_notifier_test.exs

```diff
@@ -175,16 +175,21 @@ defmodule MailerNotifierTest do
           Regex.scan(~r/<ul.+?>(.)+?<\/ul>/s, body)
           |> Enum.at(4)
 
-        [first_stack_line | _] =
+        stacktrace_list =
           Regex.scan(~r/<li>(.)+?<\/li>/s, stacktrace_list)
           |> Enum.map(&Enum.at(&1, 0))
 
-        [file, exception] =
-          Regex.scan(~r/<span.*>(.+)<\/span>/, first_stack_line)
-          |> Enum.map(&Enum.at(&1, 1))
-
-        assert "test/mailer_notifier_test.exs:18" = file
-        assert "Elixir.MailerNotifierTest.TestController.index/2" = exception
+        assert [
```

Comment:
Maybe we can test only some lines but it was for the sake of verifying that everything is being included